assets/src/edit-story/elements/text/edit.js

```diff
@@ -121,9 +121,14 @@ function TextEdit({
   const wrapperRef = useRef(null);
   const textBoxRef = useRef(null);
   const editorRef = useRef(null);
+  const boxRef = useRef();
   const contentRef = useRef();
   const editorHeightRef = useRef(0);
 
+  useEffect(() => {
```

Comment:
Can you add a comment why this is done. That's an acceptable fix for a loop. We just need to make it clear. Also, wouldn't we need to do the same for `height`? It's also updated right next to `x` and `y`.

Reply:
Correct. `x`, `y`, `height`, `rotationAngle` should be included. `updateContent` needs to be called on unmount. To prevent unnecessary `updateContent` calling move the dependency values to `boxRef`